platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -601,13 +601,9 @@ private Response showLoginPage(
             .getSamlpResponse(relayState, authnRequest, samlpResponse, cookie, template);
       } else {
         LOGGER.debug("Building the JSON map to embed in the index.html page for login.");
-        Document doc = DOMUtils.createDocument();
-        doc.appendChild(doc.createElement("root"));
-        String authn = DOM2Writer.nodeToString(OpenSAMLUtil.toDom(authnRequest, doc, false));
-        String encodedAuthn = RestSecurity.deflateAndBase64Encode(authn);
         responseMap.put(PKI, hasCerts);
         responseMap.put(GUEST, guestAccess);
-        responseMap.put(SAML_REQ, encodedAuthn);
+        responseMap.put(SAML_REQ, samlRequest);
```

Comment:
Do you think we should leave a inline comment that the *original* samlRequest must be used as decoding/re-encoding will cause issues? Dunno if we will ever have anyone try to refactor code in here but it'd be a shame to have them make that mistake.

Reply:
That's a good idea. I'll make that change. I'll add the spec too.